### ISSUE TITLE
V 1.66 not working at all on 10.43.58

### ISSUE THREAD

**Issue by Tim3tripp3r** [pylerSM/YouTubeBackgroundPlayback#83]
Hi, I have had your app working perfectly however since I updated to You Tube 10.43.58 and v 1.66 it just won't work at all for me both with or without headphones.  My details are: Moto X Play on Lollipop 5.1.1, rooted, Xposed framework 74, Installer 3.0 alpha 4. Everything is the same as far as Xposed goes just the updates to You Tube &  your app.

**Comment by esgie** [comment]
Not working at all? Gee weird.
1) Can you post a logcat?
2) Didn't you forget to check the module in xposed installer?
3) Did you reboot?

**Comment by Tim3tripp3r** [comment]
Hi,  I got it sorted out. I am a fairly experienced user but sometimes it the little details that trip you up. I use xinstaller (I am pretty sure you know what that is) and ticked "enable external storage" thinking that it would give you the option NOT make it the default install location. So anyway I was in the shower pondering life & the reasons why your app was misbehaving and thought "wonder if that app is installed on the external SD" so I checked and sure enough it was. Moved it over to phone and bam all is well. Thank you for the great app and the timely replies to help me. Another mystery solved. :) 

On Nov 3, 2015, esgie <notifications@github.com> wrote:
>Not working at all? Gee weird.
>1) Can you post a logcat?
>2) Didn't you forget to check the module in xposed installer?
>3) Did you reboot?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/pylerSM/YouTubeBackgroundPlayback/issues/83#issuecomment-153444136

Regards, Dale S. 

Connected by Motorola Moto X Play on the FIDO network
